src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -523,6 +521,26 @@ OAUTH2_ACCESS_TOKEN_URL_CONFIG, getString(OAUTH2_ACCESS_TOKEN_URL_CONFIG),
                                     + " = " + AuthorizationType.OAUTH2);
                 }
                 break;
+            case APIKEY:
```

Comment:
everything is now duplicated with OAUTH2. This could be simplified with:
```
        var authorizationType = authorizationType();
        switch (authorizationType) {
            case STATIC:
               ...
            case OAUTH2:
            case APIKEY:
                if(oauth2AccessTokenUri() == null) {
                    throw new ConfigException(
                            OAUTH2_ACCESS_TOKEN_URL_CONFIG, getString(OAUTH2_ACCESS_TOKEN_URL_CONFIG),
                            "Must be present when " + HTTP_HEADERS_CONTENT_TYPE_CONFIG
                                    + " = " + authorizationType);
                }
...
```